features/oyas.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Support for .oya files
+
+Background:
+   Given I'm in project dir
+
+Scenario: It loads values from *.oya
+  Given file ./Oyafile containing
+    """
+    Project: Secrets
+    Values:
+      foo: apple
+
+    showValues: |
+      echo ${Oya[foo]}
+      echo ${Oya[bar]}
+      echo ${Oya[baz]}
+    """
+  And file ./values1.oya containing
+    """
+    bar: banana
+    """
+  And file ./values2.oya containing
+    """
+    baz: orange
+    """
+  When I run "oya run showValues"
+  Then the command succeeds
+  And the command outputs to stdout
+  """
+  apple
+  banana
+  orange
+
+  """
+
+Scenario: It correctly merges values, processing *.oya alphabetically
```

Comment:
I'd love a test for `1_values.oya`, `10_values.oya`, `2_values.oya` so that we ordering is explicitly defined ;)